### ISSUE TITLE
Not able to use controller buttons after writing over 71 lines of code.

### ISSUE THREAD

**Issue by alaahussein94** [tylermcginnis/javascriptvisualizer#32]
When I write over 71 lines of code the controller buttons used to run and pause my code disappears. I believe it could be a styling issue because as I write my code the the controller buttons keep getting smaller until they are not clickable. 

I am using a Chrome Version 69.0.3497.100 (Official Build) (64-bit).

**Comment by labtocat** [comment]
@alaahussein94 I am not sure with the context out here, can you confirm that this is not a duplicate of issue #23 ? If not please provide screenshots for additional debugging.

**Comment by alaahussein94** [comment]
Seems to be the same as issue #23.

**Comment by labtocat** [comment]
@alaahussein94 we have a PR #25 raised for the same. once @tylermcginnis will be getting this live at the earliest.

**Comment by tylermcginnis** [comment]
Yeah I'll get to it soon. Busy week this week but I'll be going through everything early next week.
